scripts/check-coding-style.sh

```diff
@@ -1,9 +1,15 @@
 #!/bin/bash
 
-script_dir=$(dirname "$(readlink -f "$0")")
-
+READLINK=$(which greadlink)
```

Comment:
The default ```readlink``` on Linux behaves differently from the one on non-Linux OS such as OS X or Solaris. The latter have _BSD_ ```readlink``` as default while Linux' ```readlink``` is _GNU_ ```readlink```. So on OS X or Solaris you have to explicitly use ```greadlink``` to get the same (_GNU_ variant) behavior as on Linux.